force-app/main/default/classes/BDI_DataImportService.cls

```diff
@@ -372,6 +402,11 @@ global with sharing class BDI_DataImportService {
     */
     global Boolean isDryRun { get; private set; }
 
+    /*******************************************************************************************************
+    * @description The source of the gift entry batch process - UI or someplace else
+    */
+    public static Boolean isFromGiftEntryUI = isExecutionSourceUserInterface();
```

Comment:
Can this drop down to:

```
@TestVisible
private static Boolean
```

I don't see it being set outside of a test class.

Reply:
It's used by the BDI_DataImport_BATCH class to get the state during instantiation.